### PR TITLE
Latest Slesarev's Data, logging, organoleptic changes

### PR DESCRIPTION
**Pull request by Zhenya-Barannik**

### CODE REVIEW

File: doc/methods.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ When pitching sample to larger one, it is more important to maintain sterility t
...
 
 ## Analysis
 
 We've determined following sd values (estimated top):
 
 test tube: 0.2 g
-40ml propagation vial: 1 g
\ No newline at end of file
+
+40ml propagation vial: 1 g
```
Comment (varovainen) [inline doc/methods.md]:
```suggestion
40 ml propagation vial: 1 g
```

File: doc/yeast-capture.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ What we focus on here is yeast purity after capture, no matter how captured stra
 
 The yeast in starting media is often in poor condition and might be stuck to the walls. To pick it into solution, we've found the following to be effective:
 
-1. Prepare approx. 50-200 ml of sterile approx. 1.040 pale wort
-2. Empty the container almost to the bottom, so that 10-50ml remains
-3. Flame the opening and add sterile wort into container; mix thoroughly
+1. Prepare approx. 50-200 ml of sterile approx. 1.040 pale wort.
+2. Empty the container almost to the bottom, so that 10-50ml remains.
```
Comment (varovainen) [inline doc/yeast-capture.md]:
```suggestion
2. Empty the container almost to the bottom, so that 10-50 ml remain.
```

File: doc/yeast-capture.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ What we focus on here is yeast purity after capture, no matter how captured stra
...
+3. Flame the opening and add sterile wort into container; mix thoroughly.
 4. Wait until fermentation is visible; yeast could be sampled at high krauzen.
 5. To sample, we again flame the opening of the bottle and pour some wort into sterile test tube. This wort could be used to streak the plate.
```
Comment (varovainen) [inline doc/yeast-capture.md]:
```suggestion
5. To sample, flame the opening of the bottle and pour some wort into sterile test tube. This wort could be used to streak the plate.
```

**Review by varovainen — CHANGES_REQUESTED** [review]

**Comment by Zhenya-Barannik** [comment]
@Slesarew @varovainen I did everything I wanted for this specific pull request. Please review.

**Comment by Zhenya-Barannik** [comment]
Btw, those files were duplicates, there were in both liquid and organoleptic folders. I deleted them in the liquid folder.
They were NOT renamed (as web interface on GitHub suggests). 
2023-06-07-3.toml
2023-06-09-6.toml
2023-06-12-2.toml

**Review by varovainen — APPROVED** [review]

**Review by Slesarew — APPROVED** [review]

**Pull request merged**